<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,3,2,8">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>时间段</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="StartTime">
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>14</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>--</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="FinishTime">
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>14</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="StatisticsButton">
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>统计</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,2,3,2,3,2,2">
     <property name="topMargin">
      <number>49</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>车流统计</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>绿灯统计</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>通行状况</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="StatisticsTable">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>6</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>7</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>8</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>统计时段</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>东西方向</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>南北方向</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>东西方向</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>南北方向</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>东西方向</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>南北方向</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
